doc/how-to-report-bugs.rst

```diff
@@ -0,0 +1,9 @@
+How to report Cabal bugs and deficiencies
```

Comment:
“bugs and feature requests” seems less reduntant

Reply:
Good point, fixed.